views/js/signup.js

```diff
@@ -1,12 +1,90 @@
 $(document).ready(function () {
   //init side bar
   $('.sidenav').sidenav();
+  M.AutoInit();
 
   var usernameBool = false;
   var emailBool = false;
   var passwordcharBool = false;
   var passwordconfirmBool = false;
 
+  // Live updates for the text fields
+  $('#username').on("input", function() {
```

Comment:
Dose not need to check both while inputing and clicke the button, should remove one

Reply:
Fixed